docs/source/tutorials.rst

```diff
@@ -8,4 +8,4 @@ Tutorials
 
   tutorials/HLS/HLS_search.ipynb
   tutorials/GEDI/SearchGEDI.ipynb
-  tutorials/GEDI/GEDI_L4A_Example.ipynb 
```

Comment:
Since it is in the 'tutorials/GEDI' directory, should we include tutorials/GEDI/GEDI_L4A_Example.ipynb in this file?

Reply:
This comment is not for the current ticket, which is about reviewing NISAR tutorial

Reply:
Just wondering, have we not pushed GEDI L4A tutorial yet?

Reply:
Or may be, it was pushed but the tutorials.rst was not updated.

Reply:
for this ticket, the updated tutorials.rst shows the following:
<img width="1489" alt="Screen Shot 2022-11-30 at 2 09 25 PM" src="https://user-images.githubusercontent.com/24756569/204919144-8875c178-b626-4f91-bbe5-6358904c2d95.png">

Reply:
> for this ticket, the updated tutorials.rst shows the following: <img alt="Screen Shot 2022-11-30 at 2 09 25 PM" width="1489" src="https://user-images.githubusercontent.com/24756569/204919144-8875c178-b626-4f91-bbe5-6358904c2d95.png">

Do we want the GEDI_L4A_Example notebook to be removed from the table of contents in the develop branch?

Reply:
GEDI_L4A_Example notebook is already merged in the development. We shouldn't remove it from `.rst`

cc: @sumantjha  @spa0002

Reply:
I think the confusion here was caused by a git workflow issue. Because NISAR wasn't done on it's own branch but just continued on the same branch as previous tutorials it introduced some issues. It might be good to create a fresh branch based on `Develop` and then add the NISAR tutorial to that so the PR is clean.